appservice/package.json

```diff
@@ -39,7 +39,7 @@
         "ms-rest-azure": "^2.4.4",
         "opn": "^5.1.0",
         "simple-git": "^1.80.1",
-        "vscode-azureextensionui": "~0.5.5",
+        "vscode-azureextensionui": "~0.7.0",
```

Comment:
The `ui/package.json` file says 0.7.1 while this is 0.7.0

Reply:
Yeah as long as the minor version matches, we should be fine. The Travis CI would never have passed if I used 0.7.1 here since I'm bumping the version of both packages in the same PR

Reply:
Scratch that, the requirement is an _at least_ 0.7.0